packages/puppeteer-core/src/node/ChromeLauncher.ts

```diff
@@ -297,3 +310,21 @@ export function getFeatures(flag: string, options: string[] = []): string[] {
       return s;
     }) as string[];
 }
+
+/**
+ * Removes all elements in-place from the given array that match the given
+ * regular expression.
+ *
+ * @internal
+ */
+export function removeMatching(array: string[], regex: RegExp): string[] {
```

Comment:
Can we create the reg-ex in this function it would be nice to have a consistent API for extracting and removing. 
Than we a better name would be `removeMatchingFlags`.

Reply:
Do you mean to change this to `string`? OK

Reply:
Done